### PR TITLE
Add support for mkdocs

### PR DESCRIPTION
**Pull request by stevemar**
As part of a larger effort of moving off of gitbook, this PR keeps existing gitbook support while adding support to deploy to github pages, specifically using the mkdocs-material library.

### CODE REVIEW

File: docs/Lab2/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Lab 2. I need to change but want none of the hassle
+# Lab 2. Make changes with Helm
```
Comment (hickeyma) [inline docs/Lab2/README.md:1]: Would have been nice to keep the old title .. less formal?

File: docs/Lab4/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
-# Lab 4. I like sharing
+# Lab 4. Share Helm Charts
```
Comment (hickeyma) [inline docs/Lab4/README.md:1]: Would have been nice to keep the old title .. less formal?

File: docs/README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Why Helm?
+# Helm 101
```
Comment (hickeyma) [inline docs/README.md:1]: Would have been nice to keep the old title .. less formal?

File: index.yaml [hunk 4]
```diff
@@ -1,28 +0,0 @@
...
-    version: 0.2.0
-  - apiVersion: v1
-    created: "2018-05-10T21:50:41.837568406-04:00"
-    description: A Helm chart to deploy Guestbook three tier web application.
-    digest: 7f186687f43008ed68e9e4a09aa4e45586e2d6ed5f85287c64bda3f5a7ea4d12
-    name: guestbook
-    urls:
-    - https://raw.githubusercontent.com/ibm/helm101/master/repo/stable/guestbook-0.1.0.tgz
-    version: 0.1.0
-generated: "2020-02-20T14:45:38.550903527Z"
```
Comment (hickeyma) [inline index.yaml]: This index file is required for using the GH Helm101 chart repo. It is used in [Lab 4](https://github.com/IBM/helm101/blob/master/tutorial/Lab4/README.md#using-charts-from-a-public-repository)

**Review by hickeyma — CHANGES_REQUESTED** [review]
Thanks for the changes @stevemar. I am not sure of the gitbook to gh pages changes but reviewed from Helm 101 perspective. Looks good apart from the removal of the index file used for helm repo.

File: docs/Lab4/README.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
 
 ## Using charts from a public repository
 
-Helm charts can be available on a remote repository or in a local environment/repository. The remote repositories can be public like [Bitnami Charts](https://github.com/bitnami/charts) or [IBM Helm Charts](https://github.com/IBM/charts), or hosted repositories like on Google Cloud Storage or GitHub. Refer to [Helm Chart Repository Guide](https://helm.sh/docs/topics/chart_repository/) for more details. You can learn more about the structure of a chart repository by examining the [chart index file](../../index.yaml) in this lab.
+Helm charts can be available on a remote repository or in a local environment/repository. The remote repositories can be public like [Bitnami Charts](https://github.com/bitnami/charts) or [IBM Helm Charts](https://github.com/IBM/charts), or hosted repositories like on Google Cloud Storage or GitHub. Refer to [Helm Chart Repository Guide](https://helm.sh/docs/topics/chart_repository/) for more details. You can learn more about the structure of a chart repository by examining the [chart index file](https://raw.githubusercontent.com/IBM/helm101/master/repo/stable/index.yaml) in this lab.
```
Comment (stevemar) [inline docs/Lab4/README.md:9]: @hickeyma here is the updated reference

File: index.yaml (hunk 4 above)
Comment (stevemar) [reply index.yaml]: @hickeyma it's also in the stable/repo directory: https://github.com/IBM/helm101/tree/master/repo/stable 

I updated Lab 4 here: https://github.com/IBM/helm101/pull/40/files#r553103989

File: docs/Lab2/README.md (hunk 1 above)
Comment (stevemar) [reply docs/Lab2/README.md:1]: As these would be used as page headers and in a table of contents i thought more formal titles were appropriate. see https://ibm.github.io/openshift101/exercise-01/ as an example on the left nav. i was worried that in a nav the less formal titles wouldn't make sense - particularly "I like sharing". :)

If this is a sticking point for you I'm happy to reverse the changes, just wanted to explain why I did that.

File: docs/Lab4/README.md (hunk 2 above)
Comment (stevemar) [reply docs/Lab4/README.md:1]: same as above

File: docs/README.md (hunk 3 above)
Comment (stevemar) [reply docs/README.md:1]: same as above

**Review by stevemar — COMMENTED** [review]
@hickeyma thanks for reviewing. we're switching repos to github pages. mkdocs is a pretty simple framework https://squidfunk.github.io/mkdocs-material/

File: docs/Lab2/README.md (hunk 1 above)
Comment (hickeyma) [reply docs/Lab2/README.md:1]: Sounds reasonable.

File: docs/Lab4/README.md (hunk 2 above)
Comment (hickeyma) [reply docs/Lab4/README.md:1]: Replied above.

File: docs/README.md (hunk 3 above)
Comment (hickeyma) [reply docs/README.md:1]: Replied above.

File: index.yaml (hunk 4 above)
Comment (hickeyma) [reply index.yaml]: That's fine in the doc but AFIK for the `helm repo add` command to work with a Helm chart repo in GH, it needs the index file in the root of the GH repo.

File: docs/Lab4/README.md (hunk 5 above)
Comment (hickeyma) [reply docs/Lab4/README.md:9]: Better with absolute path.

**Review by hickeyma — CHANGES_REQUESTED** [review]
Changes are good bar still needs the index file in the gh repo root.

File: index.yaml (hunk 4 above)
Comment (stevemar) [reply index.yaml]: ohhh that's something i didn't know about. thank you! i'll revert the change and add a comment to the top of the file.

File: index.yaml [hunk 6]
```diff
@@ -1,3 +1,6 @@
+# Do not delete this file even though there is a duplicate in the repo/stable folder.
+# For the "helm repo add" command to work with a Helm chart repo in GH, the index
+# file must reside in the root of the repo.
```
Comment (stevemar) [inline index.yaml:3]: @hickeyma added a comment here so no one tries to delete it in the future :)

Comment (hickeyma) [reply index.yaml:3]: That's a good idea.

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @stevemar

**Comment by hickeyma** [comment]
@stevemar One small thing before merging, why change `tutorial` path to `docs`?

**Comment by stevemar** [comment]
@hickeyma the material for mkdocs framework depends on a `docs` folder: https://squidfunk.github.io/mkdocs-material/creating-your-site/

```
.
├─ docs/
│  └─ index.md
└─ mkdocs.yml
```

Note that since the `.gitbook.yaml` file was updated to also read from the `docs` folder both the gitbook and gh pages site will be available.

**Comment by hickeyma** [comment]
Ah yes, see that from the framework doc.

**Pull request merged**